modules/bridge_inner/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "bridge_inner"
```

Comment:
let's strict to naming convention with `iroha_bridge`

Reply:
All things in this repo are related to Iroha, so is it not obvious?

Reply:
fixed